docs/content/latest/yugabyte-platform/alerts-monitoring/slow-queries-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+headerTitle: Slow Queries dashboard
```

Comment:
@yb-andrew Please include these pointers?
1. Search and query language supported
2. On ramp from other pages 

Similar to what we have documented here (point 3,4,5) - https://docs.yugabyte.com/latest/yugabyte-platform/alerts-monitoring/live-queries-dashboard/